api/s2n.h

```diff
@@ -1101,6 +1101,11 @@ extern int s2n_config_add_ticket_crypto_key(struct s2n_config *config,
                                             const uint8_t *name, uint32_t name_len,
                                             uint8_t *key, uint32_t key_len,
                                             uint64_t intro_time_in_seconds_from_epoch);
+
```

Comment:
We've started using Doxygen since this PR was started... new API needs some comments.